qldbdriver/session_management_test.go

```diff
@@ -51,7 +51,7 @@ func TestSessionManagementIntegration(t *testing.T) {
 	})
 
 	t.Run("Get session when pool doesnt have session and has not hit limit", func(t *testing.T) {
-		driver, err := testBase.getDriver(ledger, 10, 4)
+		driver, err := testBase.getDriver(*testBase.ledgerName, 10, 4)
```

Comment:
seems odd this even takes a ledger name because we already set one for `testBase`. I can change that when i implement faster delete